### ISSUE TITLE
Signals not begin sent; Crash on project load; GDscript interpreter loosing mind

### ISSUE THREAD

**Issue by unfa** [godotengine/godot#63680]
### Godot version

v4.0.alpha.calinou (9094262a6b9f3d7fc5aa8a8dd582ec1058bfce71)

### System information

Arch Linux, KDE Plasma, Radeon RX6800XT, radeonpro driver, Vulkan

### Issue description

I have had trouble with a signal not being emitted in my code.
I have tried everything, changed it's name , triple checked all my code with debug prints and made absolutely sure the signal is properly connected, but nothing worked. I've decided to restart Godot editor as it seemed like an engine bug.

After restarting I got a crash on project load. So I did what I always do - I deleted the `.godot` folder in my game project, downloaded the newest alpha build and tried again - but it didn't help.

Here's the backtrace that Godot produced:

```
================================================================
handle_crash: Program crashed with signal 11
Engine version: Godot Engine v4.0.alpha.calinou (9094262a6b9f3d7fc5aa8a8dd582ec1058bfce71)
Dumping the backtrace. Please include this when reporting the bug to the project developer.
[1] /usr/lib/libc.so.6(+0x3e8e0) [0x7f8d2ec478e0] (??:0)
[2] GDScriptDataType::operator PropertyInfo() const (??:0)
[3] GDScript::_get_script_method_list(List<MethodInfo, DefaultAllocator>*, bool) const (??:0)
[4] GDScript::_update_doc() (??:0)
[5] GDScript::reload(bool) (??:0)
[6] GDScriptCache::get_full_script(String const&, Error&, String const&) (??:0)
[7] ResourceFormatLoaderGDScript::load(String const&, String const&, Error*, bool, float*, ResourceFormatLoader::CacheMode) (??:0)
[8] ResourceLoader::_load(String const&, String const&, String const&, ResourceFormatLoader::CacheMode, Error*, bool, float*) (??:0)
[9] ResourceLoader::_thread_load_function(void*) (??:0)
[10] ResourceLoader::load(String const&, String const&, ResourceFormatLoader::CacheMode, Error*) (??:0)
[11] ResourceLoaderText::load() (??:0)
[12] ResourceFormatLoaderText::load(String const&, String const&, Error*, bool, float*, ResourceFormatLoader::CacheMode) (??:0)
[13] ResourceLoader::_load(String const&, String const&, String const&, ResourceFormatLoader::CacheMode, Error*, bool, float*) (??:0)
[14] ResourceLoader::_thread_load_function(void*) (??:0)
[15] ResourceLoader::load(String const&, String const&, ResourceFormatLoader::CacheMode, Error*) (??:0)
[16] ResourceLoaderText::load() (??:0)
[17] ResourceFormatLoaderText::load(String const&, String const&, Error*, bool, float*, ResourceFormatLoader::CacheMode) (??:0)
[18] ResourceLoader::_load(String const&, String const&, String const&, ResourceFormatLoader::CacheMode, Error*, bool, float*) (??:0)
[19] ResourceLoader::_thread_load_function(void*) (??:0)
[20] ResourceLoader::load(String const&, String const&, ResourceFormatLoader::CacheMode, Error*) (??:0)
[21] GDScriptAnalyzer::reduce_preload(GDScriptParser::PreloadNode*) (??:0)
[22] GDScriptAnalyzer::reduce_expression(GDScriptParser::ExpressionNode*, bool) (??:0)
[23] GDScriptAnalyzer::reduce_call(GDScriptParser::CallNode*, bool, bool) (??:0)
[24] GDScriptAnalyzer::reduce_expression(GDScriptParser::ExpressionNode*, bool) (??:0)
[25] GDScriptAnalyzer::resolve_variable(GDScriptParser::VariableNode*) (??:0)
[26] GDScriptAnalyzer::resolve_node(GDScriptParser::Node*) (??:0)
[27] GDScriptAnalyzer::resolve_suite(GDScriptParser::SuiteNode*) (??:0)
[28] GDScriptAnalyzer::resolve_function_body(GDScriptParser::FunctionNode*) (??:0)
[29] GDScriptAnalyzer::resolve_class_body(GDScriptParser::ClassNode*) (??:0)
[30] GDScriptAnalyzer::resolve_program() (??:0)
[31] GDScriptAnalyzer::analyze() (??:0)
[32] GDScript::reload(bool) (??:0)
[33] GDScriptCache::get_full_script(String const&, Error&, String const&) (??:0)
[34] ResourceFormatLoaderGDScript::load(String const&, String const&, Error*, bool, float*, ResourceFormatLoader::CacheMode) (??:0)
[35] ResourceLoader::_load(String const&, String const&, String const&, ResourceFormatLoader::CacheMode, Error*, bool, float*) (??:0)
[36] ResourceLoader::_thread_load_function(void*) (??:0)
[37] ResourceLoader::load(String const&, String const&, ResourceFormatLoader::CacheMode, Error*) (??:0)
[38] ResourceLoaderText::load() (??:0)
[39] ResourceFormatLoaderText::load(String const&, String const&, Error*, bool, float*, ResourceFormatLoader::CacheMode) (??:0)
[40] ResourceLoader::_load(String const&, String const&, String const&, ResourceFormatLoader::CacheMode, Error*, bool, float*) (??:0)
[41] ResourceLoader::_thread_load_function(void*) (??:0)
[42] ResourceLoader::load(String const&, String const&, ResourceFormatLoader::CacheMode, Error*) (??:0)
[43] GDScriptAnalyzer::reduce_preload(GDScriptParser::PreloadNode*) (??:0)
[44] GDScriptAnalyzer::reduce_expression(GDScriptParser::ExpressionNode*, bool) (??:0)
[45] GDScriptAnalyzer::reduce_call(GDScriptParser::CallNode*, bool, bool) (??:0)
[46] GDScriptAnalyzer::reduce_expression(GDScriptParser::ExpressionNode*, bool) (??:0)
[47] GDScriptAnalyzer::reduce_assignment(GDScriptParser::AssignmentNode*) (??:0)
[48] GDScriptAnalyzer::reduce_expression(GDScriptParser::ExpressionNode*, bool) (??:0)
[49] GDScriptAnalyzer::resolve_node(GDScriptParser::Node*) (??:0)
[50] GDScriptAnalyzer::resolve_suite(GDScriptParser::SuiteNode*) (??:0)
[51] GDScriptAnalyzer::resolve_if(GDScriptParser::IfNode*) (??:0)
[52] GDScriptAnalyzer::resolve_node(GDScriptParser::Node*) (??:0)
[53] GDScriptAnalyzer::resolve_suite(GDScriptParser::SuiteNode*) (??:0)
[54] GDScriptAnalyzer::resolve_function_body(GDScriptParser::FunctionNode*) (??:0)
[55] GDScriptAnalyzer::resolve_class_body(GDScriptParser::ClassNode*) (??:0)
[56] GDScriptAnalyzer::resolve_program() (??:0)
[57] GDScriptAnalyzer::analyze() (??:0)
[58] GDScript::reload(bool) (??:0)
[59] GDScriptCache::get_full_script(String const&, Error&, String const&) (??:0)
[60] ResourceFormatLoaderGDScript::load(String const&, String const&, Error*, bool, float*, ResourceFormatLoader::CacheMode) (??:0)
[61] ResourceLoader::_load(String const&, String const&, String const&, ResourceFormatLoader::CacheMode, Error*, bool, float*) (??:0)
[62] ResourceLoader::_thread_load_function(void*) (??:0)
[63] ResourceLoader::load(String const&, String const&, ResourceFormatLoader::CacheMode, Error*) (??:0)
[64] ResourceLoaderText::load() (??:0)
[65] ResourceFormatLoaderText::load(String const&, String const&, Error*, bool, float*, ResourceFormatLoader::CacheMode) (??:0)
[66] ResourceLoader::_load(String const&, String const&, String const&, ResourceFormatLoader::CacheMode, Error*, bool, float*) (??:0)
[67] ResourceLoader::_thread_load_function(void*) (??:0)
[68] ResourceLoader::load(String const&, String const&, ResourceFormatLoader::CacheMode, Error*) (??:0)
[69] ResourceLoaderText::load() (??:0)
[70] ResourceFormatLoaderText::load(String const&, String const&, Error*, bool, float*, ResourceFormatLoader::CacheMode) (??:0)
[71] ResourceLoader::_load(String const&, String const&, String const&, ResourceFormatLoader::CacheMode, Error*, bool, float*) (??:0)
[72] ResourceLoader::_thread_load_function(void*) (??:0)
[73] ResourceLoader::load(String const&, String const&, ResourceFormatLoader::CacheMode, Error*) (??:0)
[74] EditorNode::load_scene(String const&, bool, bool, bool, bool, bool) (??:0)
[75] EditorNode::_sources_changed(bool) (??:0)
[76] void call_with_variant_args_helper<EditorNode, bool, 0ul>(EditorNode*, void (EditorNode::*)(bool), Variant const**, Callable::CallError&, IndexSequence<0ul>) (??:0)
[77] void call_with_variant_args<EditorNode, bool>(EditorNode*, void (EditorNode::*)(bool), Variant const**, int, Callable::CallError&) (??:0)
[78] CallableCustomMethodPointer<EditorNode, bool>::call(Variant const**, int, Variant&, Callable::CallError&) const (??:0)
[79] Callable::callp(Variant const**, int, Variant&, Callable::CallError&) const (??:0)
[80] Object::emit_signalp(StringName const&, Variant const**, int) (??:0)
[81] Error Object::emit_signal<bool>(StringName const&, bool) (??:0)
[82] EditorFileSystem::_notification(int) (??:0)
[83] EditorFileSystem::_notificationv(int, bool) (??:0)
[84] Object::notification(int, bool) (??:0)
[85] SceneTree::_notify_group_pause(StringName const&, int) (??:0)
[86] SceneTree::process(double) (??:0)
[87] Main::iteration() (??:0)
[88] OS_LinuxBSD::run() (??:0)
[89] /data/Projects/Games/Liblast/Repo/Godot/2022-07-30/godot(main+0x17b) [0x55bc16ed6120] (??:0)
[90] /usr/lib/libc.so.6(+0x29290) [0x7f8d2ec32290] (??:0)
[91] /usr/lib/libc.so.6(__libc_start_main+0x8a) [0x7f8d2ec3234a] (??:0)
[92] /data/Projects/Games/Liblast/Repo/Godot/2022-07-30/godot(_start+0x2a) [0x55bc16ed5eea] (??:0)
-- END OF BACKTRACE --
================================================================
```

Here is a full terminal output when loading the project:

https://zerobin.net/?422010bd570008cc#HLwHaFhiGzhRLMeHhVk+Vvji/jTrL+AVf3bdu3IQKsI=

It seems like GDScript interpreter has totally bugged out, not recognizing Nodes and their methods.

### Steps to reproduce

???

### Minimal reproduction project

???

**Comment by rune-scape** [comment]
this looks like a crash fixed by #68717

**Comment by akien-mga** [comment]
Assuming fixed. Please comment if it's still reproducible.
